adam/language_specific/english/english_language_generator.py

```diff
@@ -220,6 +209,63 @@ def _translate_object_to_noun(
 
             return dependency_node
 
+        def add_determiner(
+            self,
+            _object: SituationObject,
+            count: int,
+            dependency_node: DependencyTreeToken,
```

Comment:
Maybe rename to `noun_dependency_node` so it is clear which node you are talking about